### ISSUE TITLE
MSVC Build broke in master on april 8th

### ISSUE THREAD

**Issue by cor3ntin** [ericniebler/range-v3#1129]
I don't have a minimal repro at this time, sorry. 

Here are the build logs
```
..\third_party\range-v3\include\meta/meta.hpp(141): error C2794: 'type': is not a member of any direct or indirect base class of 'ranges::v3::adaptor_cursor<BaseIter,Adapt>::basic_adaptor_mixin_'
..\third_party\range-v3\include\range/v3/view_adaptor.hpp(266): note: see reference to alias template instantiation 'meta::_t<ranges::v3::adaptor_cursor<BaseIter,Adapt>::basic_adaptor_mixin_>' being compiled
..\third_party\range-v3\include\range/v3/view_adaptor.hpp(429): note: see reference to class template instantiation 'ranges::v3::adaptor_cursor<BaseIter,Adapt>' being compiled
```

**Comment by CaseyCarter** [comment]
Will be fixed by #1128.

**Comment by CaseyCarter** [comment]
Note to self: Two hours is not enough; must submit bug fixes 5-6 hours before the bug is reported to allow time for CI to complete.
